clash-prelude/Setup.hs

```diff
@@ -9,14 +9,6 @@ import Distribution.Extra.Doctest ( defaultMainWithDoctests )
 main :: IO ()
 main = defaultMainWithDoctests "doctests"
 #else
-#ifdef MIN_VERSION_Cabal
--- If the macro is defined, we have new cabal-install,
--- but for some reason we don't have cabal-doctest in package-db
---
--- Probably we are running cabal sdist, when otherwise using new-build
--- workflow
-import Warning ()
-#endif
```

Comment:
Building a package fails with this in (module `Warning` can't be found). No idea if it's useful to keep?

Reply:
I encountered this when I was looking at `cabal-rpm` and was going to ask you. Is there anything useful if you `git blame` it?

Reply:
Hmm, Christiaan recently added it. @christiaanb , thoughts?

Reply:
Oops. I forgot to add `Warning.hs`. Anyhow, you can also replace that `import` by:

```
#warning You are configuring this package without cabal-doctest installed. \
         The doctests test-suite will not work as a result. \
         To fix this, install cabal-doctest before configuring.
```